DEVELOPMENT.md

```diff
@@ -0,0 +1,18 @@
+Development Setup
+===============
+This document provides recommendations for setting up a develpment environment
+to facilitate reproducible research.
+# Github Rulesets
+Github rulesets define rules for interacting with a repository.  They are
+intended to improve code quality through better code review and merging
+practices.
+
+A Github ruleset can be setup by importing a ruleset JSON file.
+To setup a ruleset follow these steps:
+* Navigate to the Github repository.
```

Comment:
I originally went to the GitHub repository on my personal computer. I think putting a note informing the user to go to GitHub on the HPC browser or providing the user with command information would be helpful.